sabnzbd/urlgrabber.py

```diff
@@ -231,6 +236,12 @@ def run(self):
                         continue
                 fetch_request.close()
 
+                if gzipped:
+                    try:
+                        data = gzip.GzipFile(fileobj=io.BytesIO(data)).read()
```

Comment:
Why not use `gzip.decompress`?
https://docs.python.org/3/library/gzip.html#gzip.decompress